### PR TITLE
Pressure power page svg

### PR DESCRIPTION
**Pull request by richsoni**
Working off https://github.com/impactmakers/climatechoice/pull/140
# ChoiceGrid
image

# Banner
image

# Config 
From homepage
image

### CODE REVIEW

File: src/components/ChoiceGrid/items.js [hunk 1]
```diff
@@ -196,6 +198,31 @@ const ChoiceGridItems = () => {
             </div>
           </Link>
         </li>
+        <li className={`${Styles.choiceGridItem} ${Styles.choiceEat}`}>
```
Comment (philsturgeon) [inline src/components/ChoiceGrid/items.js]: This probably needs changing to not be Eat. Can you make something?

File: src/components/ChoiceGrid/items.js [hunk 2]
```diff
@@ -196,6 +198,31 @@ const ChoiceGridItems = () => {
...
+          <Link
+            className={Styles.choiceContainer}
+            to="/pressure-those-in-power"
+          >
+            <div className={Styles.choiceHeader}>
+              <div className={Styles.choiceIcon}>
+                <img src={EatIcon} alt="Politician" />
```
Comment (philsturgeon) [inline src/components/ChoiceGrid/items.js]: Also eat!

File: src/pages/pressure-those-in-power/index.js [hunk 3]
```diff
@@ -0,0 +1,546 @@
...
+import HowIcon from "../../components/Images/Icons/HowIcon.svg";
+import EarthyShare from "../../components/EarthyShare";
+import SuccessBanner from "../../components/SuccessBanner";
+
+const PressureThoseInPower = () => (
+  <Layout>
+    <SEO
+      title="Pressure those in power to help prevent climate change"
+      description="Climate change is an urgent crisis that cannot be held separate from politics for real change to occur"
+      image="https://climatechoice.co/images/Seo/eat.jpg"
```
Comment (richsoni) [inline src/pages/pressure-those-in-power/index.js]: also this is eat.jpg.  yeah @philsturgeon ill update all of thesse

**Comment by richsoni** [comment]
image

**Comment by philsturgeon** [comment]
Amazing Rich, thank you!! I’ll get this shared out on Monday! 🥳

**Comment by richsoni** [comment]
👍

**Pull request merged**